larsim/IonizationScintillation/IonAndScint_module.cc

```diff
@@ -107,6 +109,10 @@ namespace larg4
         {
             fISAlg = new ISCalcSeparate();
         }
+        else if(calcTag.label().compare("Correlated") == 0)
+        {
+            fISAlg = new ISCalcCorrelated();
+        }
```

Comment:
This will work just fine and do what is required.  Are you willing to polish some of this design (which was put in place before your contributions here)?  If so, I would recommend something like:
```.cpp
if (calcTag.label() == "NEST") {
  fISAlg = std::make_unique<ISCalcNESTLAr>(fEngine);
}
else if (calcTag.label() == "Separate") {
  ...
}
...
```
With this approach, you can directly compare the `label()` with the string literal using `==`.  Also, the `fISAlg` object would be of type `std::unique_ptr<ISCalc>`, which would mean that the `delete fISAlg` statement later on would be unnecessary as that would happen automatically whenever the class is destroyed.

Lastly, I argue all of the `beginJob` functionality can be moved to the constructor.  No reason to have to implement a separate function to initialize the class.

I know these design issues are not particularly related to your PR.  But while you were in this "neck of the woods," I thought you might be willing to consider it.